src/pages/SearchPage/components/SearchObjectEntry/SearchObjectEntry.module.css

```
@@ -1,6 +1,4 @@
-import styled from 'styled-components';
-
-export const Entry = styled.div`
+.entry {
   display: flex;
   align-items: center;
   grid-gap: 1.2rem;
```

Stylelint (17.7.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)